.github/actions/security-hub-scan/action.yaml

```diff
@@ -0,0 +1,59 @@
+name: security-hub-scan
+description: Scans a zarf package with uds-security-hub
+
+inputs:
+  path:
+    description: Path to the zarf package to scan
+    required: true
+  format:
+    description: Scan output format
+    default: json
+  version:
+    description: Version of uds-security-hub to use
+    default: v0.1.0
+
+outputs:
+  cve-scan:
+    description: A filename that includes the results in input.format
+    value: ${{ steps.scan.outputs.cve-scan }}
+  cosign-bundle:
+    description: A file that includes the cosign bundle for verifying authenticity
+    value: ${{ steps.cosign.outputs.cosign-bundle }}
+
+runs:
+  using: composite
+  steps:
+    - name: run uds security-hub:scan
+      id: scan
+      run: |
+        OUTPUT_FILE=cve-scan.${{ inputs.format }}
+        uds run security-hub:scan \
+          --set PKG_PATH=${{ inputs.path }} \
+          --set OUTPUT_FILE=${OUTPUT_FILE} \
+          --set OUTPUT_FORMAT=${{ inputs.format }} \
+          --set VERSION=${{ inputs.version }}
+
+        echo "cve-scan=${OUTPUT_FILE}" >> "$GITHUB_OUTPUT"
+      shell: bash
+
+    - name: install cosign
+      uses: sigstore/cosign-installer@4959ce089c160fddf62f7b42464195ba1a56d382 # v3.6.0
```

Comment:
We can likely assume this happens in env setup per the other comment

Reply:
done